.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Test
+name: Ruby SDK
```

Comment:
major: this is a general test workflow, not ruby-specific one. GitHub kinda messes up Actions tab on workflow renames so it's better to leave it as is

Reply:
@rasendubi Do you want us to have a separate flow for each language? I see you created a `python.yml` alongside `ci.yml` and `publish.yml`.

Reply:
I don't have a strong preference. Python workflow file was largely auto-generated, so it made sense to pull all python stuff into one file (including release). With a large portion of Ruby pushed to a new workflow, it makes sense to have a workflow per SDK (and remove `ci` and `publish` files later)